keyboards/ortho5by12/readme.md

```diff
@@ -0,0 +1,62 @@
+# ortho5by12 // Through Hole
+
+a plaid style 5x12 design with two indicator leds.
```

Comment:
```suggestion
A Plaid-style 5x12 design with two indicator LEDs.
```

Reply:
That git command won't work for me because i submitted it through the web browser. if you can fix it yourself then do so and i'll approve it.